tests/ts_cluster_capabilities_SUITE.erl

```diff
@@ -0,0 +1,222 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2016 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(ts_cluster_capabilities_SUITE).
+
+-compile(export_all).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+%%--------------------------------------------------------------------
+%% COMMON TEST CALLBACK FUNCTIONS
+%%--------------------------------------------------------------------
+
+suite() ->
+    [{timetrap,{minutes,10}}].
+
+init_per_suite(Config) ->
+    Config.
+
+end_per_suite(_Config) ->
+    ok.
+
+init_per_group(_GroupName, Config) ->
+    Config.
+
+end_per_group(_GroupName, _Config) ->
+    ok.
+
+init_per_testcase(_TestCase, Config) ->
+    %% tear down the whole cluster before every test
+    rtdev:setup_harness('_', '_'),
+    ct:pal("TEST CASE ~p", [_TestCase]),
+    Config.
+
+end_per_testcase(_TestCase, _Config) ->
+    ok.
+
+groups() ->
+    [].
+
+all() -> 
+    rt:grep_test_functions(?MODULE).
+
+-define(TS_VERSION_CURRENT, "current").
+
+%% git checkout riak_ts-1.3.1 && make locked-deps
+-define(TS_VERSION_1_3, "ts_1.3.1").
```

Comment:
It is 1.3.1 specific, this version doesn't have the select record capability.  At some point, some of these tests will die because there is no chance of having a mixed version cluster with 1.3.1 in.  Happy to rename to whatever makes it easier.